### PR TITLE
Feature/image

### PR DESCRIPTION
**Pull request by sanjitsarkar**
Added styling and responsiveness for image

### CODE REVIEW

File: css/components/avatar/index.css [hunk 1]
```diff
@@ -0,0 +1,94 @@
+.avatar {
+  border-radius: 50%;
+  border: 0.3rem solid var(--light-color);
```
Comment (bossmonkey98) [inline css/components/avatar/index.css:3]: Use px here

File: css/components/avatar/index.css [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  height: 5rem;
+}
+
+.avatar-sm {
+  width: 4rem;
+  height: 4rem;
+}
+
+.avatar-xsm {
+  border-width: 0.15rem;
```
Comment (bossmonkey98) [inline css/components/avatar/index.css:65]: Use px for values less than 10px

File: css/components/avatar/index.css [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    transition: all .3s cubic-bezier(0.175, 0.885, 0.32, 1.275);
+    box-shadow: var(--bx-sh-3); */
+}
+.avatar-badge {
+  right: 0%;
+  top: 0%;
+  position: absolute;
+  border-radius: 50%;
+  min-width: 2rem;
+  padding: 0.25rem;
```
Comment (bossmonkey98) [inline css/components/avatar/index.css:83]: Use px

File: css/components/badge/index.css [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+}
+.badge {
+  right: -5%;
+  top: -5%;
+  position: absolute;
+  display: grid;
+  place-content: center;
+  border-radius: 50%;
+  width: 1.5rem;
+  padding: 0.25rem;
```
Comment (bossmonkey98) [inline css/components/badge/index.css:12]: use px

File: css/components/button/index.css [hunk 5]
```diff
@@ -0,0 +1,152 @@
+button,
+input[type="submit"] {
+  cursor: pointer;
+  outline: none;
+  border: none;
+  padding: 0.5rem 0.6rem;
```
Comment (bossmonkey98) [inline css/components/button/index.css:6]: same thing

File: css/components/button/index.css [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+.btn {
+  text-decoration: none;
+  cursor: pointer;
+  display: inline-block;
+  border: 0;
+
+  background-color: transparent;
+  box-shadow: var(--bx-sh-1);
+  transform: scale(1);
+  font-size: 0.9rem;
```
Comment (bossmonkey98) [inline css/components/button/index.css:21]: +1

File: css/components/card/index.css [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+  position: absolute;
+  top: 0.5rem;
+  left: 0.5rem;
+  font-size: 0.9rem;
+  color: var(--light-color);
+  padding: 0.3rem 0.6rem;
+  border-radius: 0.3rem;
+}
+.card .card-header img {
+  border-radius: 0.25rem;
```
Comment (bossmonkey98) [inline css/components/card/index.css:48]: +1

File: css/components/card/index.css [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+
+.card .card-bottom {
+  display: flex;
+  height: 100%;
+  flex-direction: column;
+  align-items: space-between;
+  justify-content: space-between;
+}
+.card .card-bottom .card-body {
+  padding: 0.4rem 0.5rem;
```
Comment (bossmonkey98) [inline css/components/card/index.css:59]: +1

**Comment by bossmonkey98** [comment]
Use px for values less than 10px

**Review by bossmonkey98 — APPROVED** [review]

File: css/components/alert/index.css [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+  gap: 1rem;
+  align-items: center;
+  padding: calc(var(--pd) * 0.5) calc(var(--pd) * 1.5);
+  border-radius: calc(var(--br-radius) * 0.5);
+  box-shadow: var(--bx-sh-3);
+  background: var(--primary-color);
+  color: var(--light-color);
+  z-index: 9999;
+  width: fit-content;
+  font-size: calc(var(--font-size));
```
Comment (anshulraheja) [inline css/components/alert/index.css:12]: No need of calc here

File: css/components/avatar/index.css [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+
+  width: 3rem;
+  height: 3rem;
+}
+.avatar-badge-container {
+  position: relative;
+  /* transform:scale(1);
+    border-radius: 50%;
+    transition: all .3s cubic-bezier(0.175, 0.885, 0.32, 1.275);
+    box-shadow: var(--bx-sh-3); */
```
Comment (anshulraheja) [inline css/components/avatar/index.css:75]: remove comments

File: css/components/button/index.css [hunk 11]
```diff
@@ -0,0 +1,152 @@
+button,
+input[type="submit"] {
+  cursor: pointer;
+  outline: none;
+  border: none;
+  padding: 0.5rem 0.6rem;
+  border-radius: 0.3rem;
+  background-color: #119195;
```
Comment (anshulraheja) [inline css/components/button/index.css:8]: use variable for color name

**Pull request merged**